### DISCUSSION TITLE
CSSBattle - open target solving - Live Stream

### DISCUSSION THREAD

**Discussion by Beowolve** [cssbattle/forum#270] [General]
I am going to host a stream next week on Tuesday starting at 13:00 when the new targets are released.
You can watch me solve the open target and I might show some ideas for other targets as well.

Join me on discord:
https://discord.gg/7HtFFBY3sW

and watch here:
https://www.twitch.tv/beoontwitch

**Comment by chinchang** [comment]
This is live, players!  https://www.twitch.tv/beoontwitch
Hop on!
![image](https://user-images.githubusercontent.com/379918/173564588-3ae4c824-8e1d-48dd-a59c-364293cfc8d9.png)

**Comment by Beowolve** [comment]
I am going to stream next week again on 11th August. starting at 16:00.

Discord Event:
https://discord.gg/ahKtmHN?event=1004651016995737610

Stream:
https://www.twitch.tv/beoontwitch

**Reply by Beowolve** [reply]
Next stream on **29.09.2022 - 18:30**

Discord Event:
https://discord.gg/bgkPrdqbdX?event=1019869814493294642

**Reply by joewinbpaul** [reply]
> Next stream on **29.09.2022 - 18:30**
> 
> Discord Event: https://discord.gg/bgkPrdqbdX?event=1019869814493294642

WhY u NoT dOiNg A lIvE sTrEaM fOr ThE cUrReNt OpEn TaRgEt (target 137)  On TwItCh???!! <p> I wAs WaItInG fOr ThAt FoR a LoNg TiMe....

**Reply by Beowolve** [reply]
Well I was sick at that day and had a cracking voice, NeXt TiMe!
Anyway I suggest joining the discord channel, then you would have known ;)

**Reply by joewinbpaul** [reply]
> Well I was sick at that day and had a cracking voice, NeXt TiMe! Anyway I suggest joining the discord channel, then you would have known ;)

oh ok

**Reply by joewinbpaul** [reply]
Seems like I was waiting for nothing....
